content/integrations/cloud_foundry.md

```diff
@@ -0,0 +1,284 @@
+---
+title: Datadog-Cloud Foundry Integration
+integration_title: Cloud Foundry
+kind: integration
+git_integration_title: cloud_foundry
+newhlevel: true
+beta: true
+---
+# Overview
+
+Any Cloud Foundry deployment can send metrics and events to Datadog that help you track the health and availability of all nodes in the deployment, monitor the jobs they run, collect metrics from the Loggregator Firehose, and more.
```

Comment:
Ok. Yeah, it was originally two sentences and I broke it into two for some reason. I'll change it to the original.